src/sentry/incidents/subscription_processor.py

```diff
@@ -129,6 +129,8 @@ def process_update(self, subscription_update):
 
         aggregation = QueryAggregations(self.alert_rule.aggregation)
         aggregation_name = query_aggregation_to_snuba[aggregation][2]
+        if len(subscription_update["values"]["data"]) > 1:
+            logger.warning("Subscription returned more than 1 row of data")
```

Comment:
It also might make some sense to include some more contextual detail here, since you're not going to have any local variables due to this being a warning and not an exception. Topic, partition, offset are probably helpful, subscription ID might also be useful.

Reply:
The kafka message info isn't available at this point, but I've included what I can.